glossary.yaml

```diff
@@ -1,5 +1,5 @@
 terms:
-  - definition: "An account is a unique identifier and a requirement to interact with an EOSIO blockchain. Unlike most other cryptocurrencies, transfers are sent to a human readable account name instead of a public key, while keys attributed to the account are used to sign transactions."
+  - definition: "An account is a unique identifier and a requirement to interact with an EOSIO blockchain. Unlike most other cryptocurrencies, transfers are sent to a human readable account name rather than a public key, while keys attributed to the account are used to sign transactions."
```

Comment:
I think according to the source below (and a few others I found related to this topic) "instead of" is better in this case.
https://www.linkedin.com/pulse/gmat-verbal-rather-than-vs-instead-prachi-pareekh/